### DISCUSSION TITLE
When a computer has multiple GPUs, how to select GPUs for training?

### DISCUSSION THREAD

**Discussion by 15762260991** [open-mmlab/mmaction2#2750] [Design]
When a computer has multiple GPUs, how to select GPUs for training?

**Comment by ahmadjaved97** [comment]
You can refer to the following document: [https://mmaction2.readthedocs.io/en/latest/user_guides/train_test.html#training-with-multiple-gpus](https://mmaction2.readthedocs.io/en/latest/user_guides/train_test.html#training-with-multiple-gpus) for training with multiple GPUs.  You can specify the number of GPUs in the following command : `bash tools/dist_train.sh ${CONFIG} ${GPUS}`
